src/explore-education-statistics-common/src/modules/charts/util/getMinorAxisSize.ts

```diff
@@ -0,0 +1,38 @@
+import { DataSetCategory } from '@common/modules/charts/types/dataSet';
+import parseNumber from '@common/utils/number/parseNumber';
+
+interface Props {
+  dataSetCategories: DataSetCategory[];
+  minorAxisSize: number | undefined;
+  minorAxisDecimals: number | undefined;
```

Comment:
Rather than undefined, can mark these as optional:

```ts
minorAxisSize?: number;
minorAxisDecimals?: number
```
This way we don't have to explicitly set these as undefined in code e.g. 

```ts
const size = getMinorAxisSize({
  dataSetCategories: testDataSetCategories,
  minorAxisUnit: '',
});
```

instead of:

```ts
const size = getMinorAxisSize({
  dataSetCategories: testDataSetCategories,
  minorAxisSize: undefined,
  minorAxisDecimals: undefined,
  minorAxisUnit: '',
});
```